### DISCUSSION TITLE
🌟 Request for JavaScript Documentation Alongside TypeScript in Valtio Documentation 📚

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [pmndrs/valtio#893] [Ideas]
Hi Valtio Team! 👋

Firstly, I'd like to express my appreciation for the fantastic work you've done with Valtio. I'm absolutely loving the experience of working with Valtio! 🌈 It's so seamless that I often forget I'm using an external library; it feels like pure React magic. ✨ Thanks for creating such a fantastic tool that blends in seamlessly with React's ecosystem. Keep up the amazing work! 🎉

Now comes to the point.
As someone who primarily works with JavaScript rather than TypeScript, I've found myself sometimes struggling to fully grasp some of the concepts and examples provided in the Valtio documentation. While TypeScript documentation is undoubtedly valuable for many users, I believe having parallel documentation in JavaScript would significantly benefit a broader audience. 🌐

Therefore, I'm reaching out to suggest the addition of JavaScript documentation alongside the existing TypeScript examples and explanations on the Valtio documentation website. This addition would not only cater to developers who prefer JavaScript but also make Valtio more accessible to those transitioning from JavaScript to TypeScript or those simply wanting to use JavaScript in their projects. 💡

Including JavaScript examples would not only improve the accessibility of Valtio but also enhance its adoption among developers from various backgrounds and skill levels. 📈

Thank you for considering my suggestion. I'm looking forward to seeing Valtio continue to evolve and grow as an essential tool in the React ecosystem. 🌱

Best regards

**Comment by dai-shi** [comment]
Hey, thanks for your feedback.
It's a valid request, but it's a bit surprising that I got this in Valtio first. I think I've never received such a request in Jotai and Zustand.

While I consider it's a nice-to-have, I wonder how much effort we should spend. We can't maintain two versions of docs. If it's just about example code snippets. I've done something in https://react-tracked.js.org where JavaScript code is automatically generated from TypeScript code upon the build process.

The biggest concern in Valtio is that the website/docs team is not very active lately. And, we would probably need to form a team again. cc: @RikuVan @roguesherlock @ScreamZ 
So, I'm wondering if you are interested in taking the read of this activity.

**Reply by dai-shi** [reply]
Wait, https://react-tracked.js.org/ is broken now. 😱 

(edit) Fixed now.
